indico/modules/categories/templates/display/base.html

```diff
@@ -18,64 +18,87 @@ <h1 class="category-title {{ category_title_classes }}">
+                            {% trans %}Navigate{% endtrans %}
+                        </button>
+                    {% endblock %}
                 </div>
-            {% endif %}
+                <div class="group">
+                    {% block cat_parent %}
+                        {% if not category.is_root %}
+                            <a class="i-button icon-arrow-up" href="{{ category.parent.url }}">
+                                {% trans %}Parent category{% endtrans %}
+                            </a>
+                        {% endif %}
+                    {% endblock %}
+
+                    {% block cat_export %}
+                        <a href="#"
+                           class="i-button icon-calendar arrow js-export-ical"
+                           title="{% trans %}Export to scheduling tool{% endtrans %}"
+                           data-id="{{ category.id }}"></a>
+                        {% with item=category, ics_url=url_for('categories.export_ical', category) -%}
+                            {% include 'categories/category_export_ical.html' %}
+                        {%- endwith %}
+                    {% endblock %}
+
+                    {% block cat_view_dropdown %}
+                        <a class="i-button icon-eye arrow js-dropdown"
+                           title="{% trans %}View{% endtrans %}"
+                           data-toggle="dropdown"></a>
+                        <ul class="i-dropdown">
+                            <li>
+                                <a href="{{ url_for('.upcoming_event', category) }}">{% trans %}Upcoming event{% endtrans %}</a>
+                            </li>
+                            <li>
+                                <a href="{{ url_for('.overview', category, period='day') }}">{% trans %}Today's events{% endtrans %}</a>
+                            </li>
+                            <li>
+                                <a href="{{ url_for('.overview', category, period='week') }}">{% trans %}Week's events{% endtrans %}</a>
+                            </li>
+                            <li>
+                                <a href="{{ url_for('.calendar', category) }}">{% trans %}Calendar{% endtrans %}</a>
+                            </li>
+                            <li>
+                                <a href="{{ url_for('.statistics', category) }}">{% trans %}Category statistics{% endtrans %}</a>
+                            </li>
+                        </ul>
+                    {% endblock %}
+
+                    {% block cat_manage %}
+                        {% if category.can_manage(session.user) %}
+                            <a href="{{ url_for('categories.manage_content', category) }}"
+                               class="i-button icon-edit"
+                               title="{% trans %}Manage category{% endtrans %}"></a>
+                        {% endif %}
+                    {% endblock %}
+
```

Comment:
I don't think this empty line was here before

Reply:
There are also other new empty lines above (between blocks, i think). I'll remove them.

Reply:
Having an empty line between two blocks seems reasonable. Having one right before the closing tag not so much

Reply:
Removed them.